lib/knife-zero/devpatch/train_connector.rb

```diff
@@ -0,0 +1,19 @@
+require 'chef/knife/bootstrap/train_connector'
+require 'knife-zero/helper'
+
+class Chef
+  class Knife
+    class Bootstrap < Knife
+      class TrainConnector
+        class_eval do
+          def connect!
+            # Force connection to establish
+            connection.wait_until_ready
+            connection.instance_variable_get(:@session).forward.remote(URI.parse(Chef::Config.chef_server_url).port, '127.0.0.1', ::Knife::Zero::Helper.zero_remote_port)
```

Comment:
__[RuboCop]__ Metrics/LineLength: Line is too long. [169/120] ([link](https://sider.review/gh/repos/23179315/pulls/125#issue-4171272))

<sub>posted by Sider</sub>